index.bs

```diff
@@ -413,65 +413,45 @@ This method takes the following parameters:
+    (as returned by [=normalizing an algorithm=]).
 1. [=list/For each=] |current| of {{cryptoParameters}}:
-    1. If <code>|current|.{{ScopedCredentialParameters/type}}</code> does not
-        contain a {{ScopedCredentialType}} supported by this implementation,
-        then [=continue=].
-    1. Let |normalizedAlgorithm| be the result of <a>normalizing an algorithm</a>
-        [[!WebCryptoAPI]], with |alg| set to
-        <code>|current|.{{algorithm}}</code> and |op| set to `"generateKey"`. If
-        an error occurs during this procedure, then [=continue=].
-    1. [=list/Append=] the pair of
-        <code>|current|.{{ScopedCredentialParameters/type}}</code> and
-        |normalizedAlgorithm| to |normalizedParameters|.
-
-1. If |normalizedParameters| is empty and {{cryptoParameters}} was not empty,
-    cancel the timer started in step 2, return [=a promise rejected with=] with
-    a {{DOMException}} whose name is "{{NotSupportedError}}", and terminate this
-    algorithm.
+    1. If <code>|current|.{{ScopedCredentialParameters/type}}</code> does not contain a {{ScopedCredentialType}} supported by
+        this implementation, then [=continue=].
+    1. Let |normalizedAlgorithm| be the result of <a>normalizing an algorithm</a> [[!WebCryptoAPI]], with |alg| set to
+        <code>|current|.{{algorithm}}</code> and |op| set to `"generateKey"`. If an error occurs during this procedure, then
+        [=continue=].
+    1. [=list/Append=] the pair of <code>|current|.{{ScopedCredentialParameters/type}}</code> and |normalizedAlgorithm| to
+        |normalizedParameters|.
+
+1. If |normalizedParameters| is empty and {{cryptoParameters}} was not empty, cancel the timer started in step 2, return
+    [=a promise rejected with=] with a {{DOMException}} whose name is "{{NotSupportedError}}", and terminate this algorithm.
 
 1. Let |clientExtensions| be a new [=list=].
-1. If the {{ScopedCredentialOptions/extensions}} member of {{options}} is
-    <a>present</a>, then [=map/for each=] |extension| → |argument| of
-    <code>{{options}}.{{ScopedCredentialOptions/extensions}}</code>:
-    1. If |extension| is not supported by this client platform, then either:
-        * [=Continue=], or
-        * Let |result| be a CBOR ([[!RFC7049]]) encoding of |extension|.
-
-            Issue(w3c/webauthn#363): Define this encoding more precisely.
-    1. Otherwise, let |result| be the result of running |extension|'s [=client
-        processing=] algorithm on |argument|. If the algorithm returned an
-        error, [=continue=].
-
-        Issue(w3c/webauthn#363): Ensure all extensions define a client
-        processing algorithm.
+1. If the {{ScopedCredentialOptions/extensions}} member of {{options}} is <a>present</a>, then [=map/for each=]
+    |extension| → |argument| of <code>{{options}}.{{ScopedCredentialOptions/extensions}}</code>:
+    1. If |extension| is not supported by this client platform, then [=continue=].
+
```

Comment:
I think this depends on the definition of what it takes to "support" an extension. If for certain extensions supporting them only requires doing some CBOR encoding, then maybe a browser that blindly does that should also be regarded as supporting the extension.